backend/pom.xml

```diff
@@ -79,6 +79,10 @@
             <groupId>io.quarkus</groupId>
             <artifactId>quarkus-hibernate-orm</artifactId>
         </dependency>
+        <dependency>
+            <groupId>io.quarkus</groupId>
+            <artifactId>quarkus-quartz</artifactId>
+        </dependency>
```

Comment:
I just removed it and it's needed at least from a technical perspective. Otherwise the Scheduled stuff doesn't compile.
Maybe you know more than I do? Is there is something better than these quartz-job?

Reply:
This should work:
```suggestion
        <dependency>
            <groupId>io.quarkus</groupId>
            <artifactId>quarkus-scheduler</artifactId>
        </dependency>
```

Reply:
aah! nice!